### DISCUSSION TITLE
CSS bug on Stars Lists

### DISCUSSION THREAD

**Discussion by jgphilpott** [community/community#19023] [Other Feature Feedback, Questions, & Ideas] [answered]
## Intro

Hey, I love the recently added lists feature on my stars tab. I did however notice one css bug on the lists view:

 - When a user has > 8 lists a row is added with a link saying "Show all lists..." rather than displaying them all by default. See screenshot one.
 - That's all fine except, when the link is clicked and all the lists are displayed the resulting table is missing a 'row separator line' where the "Show all lists..." link used to be. See screenshot two.
 - This brakes the visual pattern that the user is expecting.

I've only tested this on chrome but if this behaviour is reproducible I think it should be considered a bug and fixed.

Thanks!

### **Screenshot One**

<img width="920" alt="Screen Shot 2022-06-21 at 10 10 43 AM" src="https://user-images.githubusercontent.com/4128208/174738464-4e6b11bd-cacf-4b81-a25b-206dd3603b5f.png">

### **Screenshot Two**

<img width="914" alt="Screen Shot 2022-06-21 at 10 11 27 AM" src="https://user-images.githubusercontent.com/4128208/174738484-d4399074-5447-4598-9b85-c25c53c1d75a.png">

**Comment by melvincwng** [comment, marked as answer]
Hi @jgphilpott, thanks for raising this issue to the community. 

For bugs, kindly raise a ticket to Github Support over [here](https://support.github.com/contact/bug-report). 

The support engineers usually reply within a couple of business days, after investigating the issue.

Hope this helps.

If you find this answer useful, please kindly 'Mark as answer' so that others know this issue has been addressed.

Thanks! 🙏

**Reply by jgphilpott** [reply]
Thanks for the link @melvincwng I opened a bug report!

**Reply by melvincwng** [reply]
Np @jgphilpott, glad to have helped!

Please kindly 'Mark as answer' so that others know this issue has been addressed.

Thanks and have a nice day 😊
